### PR TITLE
Chandani_Assignment_9

### PR DESCRIPTION
**Pull request by chandni0604**
image

Output after updates in program (Incorporated review comments ) 
image

### CODE REVIEW

**Review by a-malvi — COMMENTED** [review]
Create a common method rather than method for all items individually.

**Review by chandni0604 — COMMENTED** [review]
Incorporated the review comments

File: src/chandani/Assignment9.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		maggieCnt = maggieQty ;
+		dosaCnt = dosaQty;
+		pouchesCnt = pouchesQty;
+		panipuriCnt = panipuriQty;
+		masalaCnt = maggieQty;
+	}
+	
+	void maggieinStockorNot(String item,int maggieAmt){
+		
+		maggieCnt = maggieCnt - maggieAmt;
```
Comment (KrishnaKTechnocredits) [inline src/chandani/Assignment9.java:21]: You should check whether request item available or not, based upon you should provide it. 
Do you agree?

**Review by KrishnaKTechnocredits — APPROVED** [review]
Conditions require, rest code looks good to e=me
For condition check others code

**Pull request merged**